superchain/configs/configs.json

```diff
@@ -1909,6 +1909,73 @@
             "UnsafeBlockSigner": "0xb830b99c95Ea32300039624Cb567d324D4b1D83C"
           }
         },
+        {
+          "Name": "Ink Sepolia",
+          "l2_chain_id": 763373,
+          "PublicRPC": "https://rpc-gel-sepolia.inkonchain.com",
+          "SequencerRPC": "https://rpc-gel-sepolia.inkonchain.com",
+          "Explorer": "https://explorer-sepolia.inkonchain.com",
+          "SuperchainLevel": 0,
+          "StandardChainCandidate": true,
+          "SuperchainTime": null,
+          "batch_inbox_address": "0x004De1914F4f17aC234f5A5Bc8a4072a231d44BF",
+          "Superchain": "sepolia",
+          "Chain": "ink",
+          "canyon_time": 0,
+          "delta_time": 0,
+          "ecotone_time": 0,
+          "fjord_time": 0,
+          "granite_time": 0,
+          "block_time": 1,
```

Comment:
I may be misunderstanding, but if `SuperchainTime` is `null` wouldn't this need to explicitly set `holocene_time` since holocene is active on ink-sepolia?

This may not be the place to fix that in the code - I just want to make sure we actually have the holocene activation set correctly.

Reply:
Correct, as is this config does not activate holocene so it needs to be fixed one way or the other.

Reply:
left a comment - suggesting to set `superchain_time = 0`.

Reply:
Done in https://github.com/ethereum-optimism/superchain-registry/pull/740/commits/44b610128821cbc64e0d972b05904c85c6300516